src/SamplesApp/SamplesApp.Skia/SamplesApp.Skia.csproj

```diff
@@ -44,6 +44,7 @@
 		<PackageReference Include="Microsoft.Graph" Version="3.12.0" />
 		<PackageReference Include="Microsoft.Identity.Client" Version="4.47.0" />
 		<PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
+		<!-- TODO: Use version 8 when compiling against .NET 8? -->
```

Comment:
I'm not sure about this one.

Reply:
I don't know either. If it works with 7.0, let's keep it that way for now.

Reply:
@jeromelaban Do you want to leave the comment or remove it?